components/sidebar/channel_navigator/channel_navigator.tsx

```diff
@@ -2,20 +2,89 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
 
-type Props = {
+import {ModalIdentifiers} from 'utils/constants';
+import QuickSwitchModal from 'components/quick_switch_modal';
+
+import * as Utils from 'utils/utils';
+import {isDesktopApp} from 'utils/user_agent';
 
+type Props = {
+    canGoForward: boolean;
+    canGoBack: boolean;
+    actions: {
+        openModal: (modalData: any) => Promise<{data: boolean}>;
+        goBack: () => void;
+        goForward: () => void;
+    };
 };
 
 type State = {
 
 };
 
 export default class ChannelNavigator extends React.PureComponent<Props, State> {
+    openQuickSwitcher = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
Does anything need to be done to reimplement this hotkey? If it's handled elsewhere in the new sidebar, maybe it should be moved here?

Reply:
If you mean the CTRL+K/Cmd+K hotkey, it's implemented elsewhere I believe. I'm pretty sure it's either in that global component we created, or a different one.

Reply:
The correct answer is that it's part of the ChannelHeader component for some reason...